index.html

```diff
@@ -331,12 +331,14 @@ <h2>Glossary</h2>
 
 
 <div class="letter_anchor" id="r">R</div>
 
-<p><dfn data-lt="resource|resources" id="def_resource" class="lint-ignore export">Resource</dfn>. In the context of W3C Internationalization documents,   a given document, file, or protocol "message" which includes both the <a href="#def_localizable_text" class="termref">localizable text</a> as well as the <a href="#def_syntactic_content" class="termref">syntactic content</a> such as identifiers surrounding or containing it. For example, in an HTML document that also has some CSS and a few <code class="kw" translate="no">script</code> elements with embedded JavaScript, the entire HTML document, considered as a file, is a resource. This term is intentionally similar to the term 'resource' as used in [[RFC3986]], although here the term is applied loosely. </p>
+<p><dfn data-lt="resource|resources" id="def_resource" class="lint-ignore export">Resource</dfn>. In the context of W3C Internationalization documents,   a given document, file, or protocol "message" which includes both the <a href="#def_localizable_text" class="termref">localizable text</a> as well as the <a href="#def_syntactic_content" class="termref">syntactic content</a> such as identifiers surrounding or containing it. For example, in an HTML document that also has some CSS and a few <code class="kw" translate="no">script</code> elements with embedded JavaScript, the entire HTML document, considered as a file, is a resource. This term is intentionally similar to the term 'resource' as used in [[RFC3986]], although here the term is applied loosely.</p>
+
+<p><dfn id="dfn_resid" data-lt="resource identifier|resource identifiers" class="lint-ignore export">Resource identifier</dfn>. A resource identifier is a compact string of characters for identifying an abstract or physical <a>resource</a>. On the Web, this mostly means various types of Universal Resource Identifiers (or <em>URIs</em>). For wire formats, [[RFC3986]] defines the structure and serialization. <em>Internationalizationed Resource Identifiers</em> (or <em>IRIs</em>) [[RFC3987]] describe how non-ASCII Unicode characters can be used in resource identifiers. The WhatWG [[URL]] specification describes how browsers handle IRIs and their mapping to URIs.</p>
```

Comment:
We should add a `class="termref"` to "resource" to make the styling more consistent.

Reply:
Internationalizationed is an interesting new word, but i don't think it's actually in the title of that document.  And the verb 'describe' should have an 's' at the end, since the subject is a document.

Reply:
This is the actual title of RFC3987: the word *internationalized* ***is*** the first word :-) https://www.rfc-editor.org/rfc/rfc3987

Reply:
added termref

Reply:
Indeed, the first word is 'Internationalized', not 'Internationalizationed'. Still needs fixing.